platform/polkadot/model.go

```diff
@@ -0,0 +1,51 @@
+package polkadot
+
+type Metadata struct {
+	BlockNum    string `json:"blockNum"`
+	BlockTime   string `json:"blockTime"`
+	ImplName    string `json:"implName"`
+	NetworkNode string `json:"networkNode"`
```

Comment:
Do you need all of the data? Can we only keep what’s needed? 


Can name BlockNum as BlockNumber? Same for others?

Reply:
simplified